nighthawk/WORKSPACE

```diff
@@ -0,0 +1,30 @@
+workspace(name = "nighthawk")
```

Comment:
Ultimately, I think we might want a root WORKSPACE, Bazel prefers to see each repo as a single tree.